Makefile

```diff
@@ -67,9 +67,12 @@ integration-win: binaries/client build-extension-types build-extensions
 	yarn build:win
 	yarn integration
 
-.PHONY: build
-build: node_modules binaries/client build-extensions
+.PHONY: fix-build-version
+fix-build-version:
 	yarn run npm:fix-build-version
+
+.PHONY: build
+build: node_modules binaries/client fix-build-version build-extensions
```

Comment:
Does this work with `make --jobs=N`?

Reply:
Oh probably not, can I run `make build-extensions` in a make command? I have never learned how to design around multiple jobs.